### PR TITLE
Generate SHA256 hashes for binaries

### PR DESCRIPTION
**Pull request by hatchan**
This is a requirement for automating our brew deployment automation

### CODE REVIEW

File: wercker.yml [hunk 1]
```diff
@@ -69,6 +69,18 @@ build:
               -output "$WERCKER_OUTPUT_DIR/latest/{{.OS}}_{{.Arch}}/sentcli"
 
             cp -r $WERCKER_OUTPUT_DIR/latest/* $WERCKER_REPORT_ARTIFACTS_DIR
+
+    # generate SHA's for all binaries
+    - script:
+        name: generate SHAs
+        code: |
+            cd $WERCKER_OUTPUT_DIR
+            for f in $(find . -type f -name sentcli); do
```
Comment (faiq) [inline wercker.yml]: is the name still sentcli?

Comment (faiq) [inline wercker.yml]: ya it is

File: wercker.yml [hunk 2]
```diff
@@ -69,6 +69,18 @@ build:
               -output "$WERCKER_OUTPUT_DIR/latest/{{.OS}}_{{.Arch}}/sentcli"
```
Comment (faiq) [inline wercker.yml]: maybe we should change the name wercker?

File: wercker.yml (hunk 1 above)
Comment (hatchan) [inline wercker.yml]: Yeah that is a separate issue. But not touching that one yet, as it is used on multiple locations

**Comment by faiq** [comment]
:+1:

File: wercker.yml (hunk 2 above)
Comment (hatchan) [inline wercker.yml]: Thats going to be a bigger PR, which covers multiple projects. For now we're just doing a `sed` on the SHA256SUM files.

**Comment by hatchan** [comment]
Thanks for the review \o/

**Pull request merged**